### ISSUE TITLE
Switching from Mapped Network Drives to Docker

### ISSUE THREAD

**Issue by kirstenkeister** [jlesage/docker-crashplan-pro#166]
I have three top-level folders of files on my Synology that I had previously mapped as three separate network drives (Files, Photos, Music), which I backed up to Crashplan using the program on my desktop computer. This created a different folder structure on the Crashplan server than what is created by default when running Crashplan as a headless client (i.e. /volume1/etc). 

I am wondering whether it is possible to create the same mappings/folder structure that I had (when using mapped network drives) with Docker such that I do not have to re-backup the entire 2TB of data. I tried several different mappings using the `-v` parameter, but was unable to get the folders mapped correctly. 

For reference, here are the two different folder structures that are created on the Crashplan server:
![file-structure_e](https://user-images.githubusercontent.com/1154129/54889470-2d550000-4e7b-11e9-91d4-e19fcc52ad58.png)

And these are how the folders are set up in File Station in the Synology DSM: 
![file-station-structure](https://user-images.githubusercontent.com/1154129/54889592-ae13fc00-4e7b-11e9-8f2b-9846e8bfe343.png)

I am currently using this volume mapping when creating the Docker container: 
`-v /volume1/docker/appdata/crashplan:/config:rw \`
`-v /volume1/:/volume1:ro \`

Is what I want to do possible? Or am I stuck re-backing up everything under volume1? 
Thanks!

**Comment by jlesage** [comment]
You don't need to have the same folder structure.  What is important is that when you first launch the Docker version of CrashPlan, you need to select the option to replace a device.  You then re-select your files under the new folder structure.  Because of deduplication, CrashPlan won't re-upload everything.

See https://github.com/jlesage/docker-crashplan-pro#taking-over-existing-backup

**Comment by kirstenkeister** [comment]
Okay gotcha. I believe I selected that when I first launched the Docker version. I'm having trouble with files constantly being scanned, but nothing is being uploaded so I wondered if that was the issue. Instead, I think perhaps that it is because I do not have enough memory allocated, even though I've broken my entire 2TB into multiple smaller sets. 

Regardless, I appreciate knowing that Crashplan is smart enough to just redirect rather than requiring re-upload.
